app/assets/javascripts/shared/rails-flash-loader.js.coffee

```diff
@@ -0,0 +1,19 @@
+angular.module('RailsFlashLoader', ['angular-flash.service'])
```

Comment:
I am all for breaking big modules up into small ones, but having a whole module for single directives/factories seems like a little bit too much granularity. IMO it doesn't improve the hierarchy because we will end up with almost as many folders as files. Module are intended as organisational units whereby the components often common dependencies. This does not mean that ALL components of a module that is inherited by another must be used by the depending module...